.travis.yml

```diff
@@ -47,6 +47,7 @@ install:
 
 # command to run tests, e.g. python setup.py test
 script:
+  - echo testing git revision $(git rev-parse HEAD) ...  # tell commit hash
```

Comment:
This seems to be irrelevant for the PR, right?

Reply:
Yes, it is. A similar line can be found in another PR.